api/lib/mno_enterprise/concerns/controllers/jpi/v1/user_access_requests_controller.rb

```diff
@@ -23,6 +23,18 @@ def last_access_request
 
   # POST /mnoe/jpi/v1/user_access_requests
   def create
+    access_duration = params.require(:access_duration)
+    user = current_user.load_required(:user_access_requests, :'user_access_requests.requester')
+    user.user_access_requests.each do |request|
```

Comment:
I would move this logic inside the class `UserAccessRequest` as a class method and then call something like
```ruby
MnoEnterprise::UserAccessRequest.approve_pending_requests(user)
```

Reply:
Ok